worker/tern_worker.js

```diff
@@ -177,7 +246,7 @@ handler.analyze = function(value, ast, callback, minimalAnalysis) {
     };
     addTernFile(this.path, value);
 
-    architectResolver.onReady(function() {
+    architectResolver && architectResolver.onReady(function() {
```

Comment:
But if it's only initialized with https://github.com/c9/c9.ide.language.javascript.tern/pull/1#discussion-diff-36980234R178, will `onReady()` always be called?